### PR TITLE
AC-306: Lint adjustment - Internationalization #layout-fixes

### PR DESCRIPTION
**Pull request by shivtej1505**

### CODE REVIEW

File: openmrs-client/src/main/res/layout/find_last_viewed_patients_row.xml [hunk 1]
```diff
@@ -56,7 +56,7 @@
         android:id="@+id/lastViewedBottomRow"
         android:orientation="horizontal"
         android:layout_marginLeft="5dp"
-        android:weightSum="6"
+        android:weightSum="9"
```
Comment (AdamGrzybkowski) [inline openmrs-client/src/main/res/layout/find_last_viewed_patients_row.xml]: This card view still doesn't look like before. The download button should be nest to the right edge
screenshot_1487838360

**Comment by shivtej1505** [comment]
@tmarzeion , PR updated!

**Comment by AdamGrzybkowski** [comment]
@shivtej1505 The screen below shows that this is not ready. The download checkbox is too high and the right margin of the second row is different from the first row(same for second layout).  I wold recommend doing it from scratches or reverting it back to the old layout.

screenshot_1488181510

**Comment by shivtej1505** [comment]
Will send updated PR soon!

**Comment by AvijitGhosh82** [comment]
Ping @shivtej1505

**Comment by shivtej1505** [comment]
PR updated!

**Comment by AdamGrzybkowski** [comment]
@shivtej1505  Please provide layout screen shots, so it will be easier for us to review it

**Comment by shivtej1505** [comment]
Screenshots:

device-2017-03-06-163832
device-2017-03-06-163945

**Comment by shivtej1505** [comment]
Screenshots:

device-2017-03-06-164244
device-2017-03-06-164306

**Comment by shivtej1505** [comment]
Screenshots added:

device-2017-03-06-164922
device-2017-03-06-164952

**Comment by AvijitGhosh82** [comment]
@AdamGrzybkowski @tmarzeion Please update your reviews?

**Comment by tmarzeion** [comment]
Good job @shivtej1505 :)
Merged

**Pull request merged**